blockchain/storage/keyvalue.go

```diff
@@ -39,6 +39,9 @@ var (
 
 	// TX_LOOKUP_PREFIX is the prefix for transaction lookups
 	TX_LOOKUP_PREFIX = []byte("l")
+
+	// LAST_PROCESSED_EVENT is the prefix for the last processed event lookups
+	LAST_PROCESSED_EVENT = []byte("last_processed_event")
```

Comment:
What does `event` mean?

Reply:
It's terminology from the rootnet staking efforts - it signifies the last processed event data for the specific smart contract

Reply:
I meant blockchain module doesn't know the term used in RootnetStaking, then supplement is needed.

Reply:
The keyvalue store is in the blockchain module (the entire storage module for that matter), that's why we need to have this key prefix in there